examples/v1/pipelineruns/pipelinerun-with-final-results.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
the CRDs in this file should be v1

Reply:
Sorry was still debugging on this as if I changed to v1, will have the error
```Error from server (BadRequest): error when creating "examples/v1/pipelineruns/pipelinerun-with-final-results.yaml": admission webhook "validation.webhook.pipeline.tekton.dev" denied the request: validation failed: invalid value: referencing a nonexistent task: spec.results[1].value
```